public-ui/src/app/components/recursive-tree/recursive-tree.component.ts

```diff
@@ -40,7 +40,7 @@ export class RecursiveTreeComponent implements OnChanges, OnDestroy {
       const item = nodes[i];
       if (item === node && i === index) {
         this.childIndex = i;
-        this.subOpened = !this.subOpened;
+        this.subOpened = true;
```

Comment:
when would this change to false on closing ?

Reply:
the component starts as false.
and when another node is clicked it sends the 'opened' var as false, which is used to render the contents in the 'folder'

Reply:
kinda hard to explain because of recursion

Reply:
ohh yeah gotcha. Was circling over there. thanks.